pkg/printers/printers.go

```diff
@@ -75,36 +75,34 @@ func GetStandardPrinter(typer runtime.ObjectTyper, encoder runtime.Encoder, deco
 			NoHeaders:        options.NoHeaders,
 			TemplateArgument: formatArgument,
 		}
-		customColumnsPrinter, matched, err := customColumnsFlags.ToPrinter(format)
-		if !matched {
-			return nil, fmt.Errorf("unable to match a name printer to handle current print options")
-		}
+		customColumnsPrinter, err := customColumnsFlags.ToPrinter(format)
 		if err != nil {
 			return nil, err
 		}
 
 		printer = customColumnsPrinter
 
-	case "wide":
-		fallthrough
-	case "":
-		humanPrintFlags := NewHumanPrintFlags(options.Kind, options.NoHeaders, options.WithNamespace, options.AbsoluteTimestamps)
-
-		// TODO: these should be bound through a call to humanPrintFlags#AddFlags(cmd) once we instantiate PrintFlags at the command level
-		humanPrintFlags.ShowKind = &options.WithKind
-		humanPrintFlags.ShowLabels = &options.ShowLabels
-		humanPrintFlags.ColumnLabels = &options.ColumnLabels
-		humanPrintFlags.SortBy = &options.SortBy
-
-		humanPrinter, matches, err := humanPrintFlags.ToPrinter(format)
-		if !matches {
-			return nil, fmt.Errorf("unable to match a printer to handle current print options")
-		}
-		if err != nil {
-			return nil, err
-		}
-
-		printer = humanPrinter
+	//case "wide":
```

Comment:
Remove?

Reply:
Yeah, left in there for review in case I had overlooked something that might break by removing this case